<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationGUI</class>
 <widget class="QMainWindow" name="CommunicationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIALOG CommunicationGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="connectedProcessesTable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>751</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <column>
     <property name="text">
      <string>Name of Process</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Provided services</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Subscribed services</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Commands</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="nameOfProcessLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of Process:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameOfProcessText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameOfCommandText">
    <property name="geometry">
     <rect>
      <x>919</x>
      <y>5</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameOfCommandLabel">
    <property name="geometry">
     <rect>
      <x>799</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Command:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameOfProvidedServiceText">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>5</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameOfProvidedServiceLabel">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>10</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Provided service:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameOfSubscribedServiceText">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>35</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameOfSubscribedServiceLabel">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Subscribed service:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>1052</x>
      <y>5</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1052</x>
      <y>35</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="pidLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pidText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>35</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>5</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>35</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
